IDE/Espressif/ESP-IDF/examples/wolfssl_client/main/client-tls.c

```diff
@@ -42,25 +40,115 @@
     #include <wolfssl/wolfcrypt/mem_track.h>
 #endif
 
-const char *TAG = "tls_client";
+#ifdef WOLFSSL_ESPWROOM32SE
+static const char* TAG = "tls_client";
+#else
+const char*  TAG = "tls_client";
+#endif
+
+static void ShowCiphers(void)
+{
+    static char ciphers[4096];
```

Comment:
This should not be static. If its static the compiler will reserve 4096 bytes globally all the time for this function.

Reply:
Changed. And only enabled the function when DEBUG_WOLFSSL is enabled.